lib/linguist/languages.yml

```diff
@@ -3753,6 +3753,19 @@ RMarkdown:
   - ".rmd"
   tm_scope: source.gfm
   language_id: 313
+RPC:
+  type: programming
+  aliases:
+  - rpcgen
+  - oncrpc
+  - xdr
+  ace_mode: c_cpp
+  extensions:
+  - ".x"
+  interpreters:
+  - rpcgen
```

Comment:
Are there actually RPC files using a `rpcgen` shebang? I [couldn't find any with GitHub searches](https://github.com/search?p=1&q=%22bin%2Frpcgen%22&type=Code&utf8=%E2%9C%93), but it's not particularly easy to search for strings with special characters.

Reply:
Not that I've ever seen, but rpcgen is the only program I know of that could.

Actually, looks like my rpcgen gets confused about the `#!` line anyway, so perhaps there's no point.

Reply:
Hm. Yes, it's probably better to remove it if it's not used.